packages/statemanager/src/ethersStateManager.ts

```diff
@@ -182,7 +182,7 @@ export class EthersStateManager implements EVMStateManagerInterface {
    * Both are represented as `0x` prefixed hex strings.
    */
   dumpStorage(address: Address): Promise<StorageDump> {
-    const storageMap = this._storageCache._lruCache?.get(address.toString())
+    const storageMap = this._storageCache.dump(address)
```

Comment:
I am generally a bit irritated how different these two (EthersSM vs DefaultSM) implementations here are, with EthersSM accessing the cache directly, while DefaultSM is not and things like that?

I think this could be more aligned in general than it is now? 🤔

Anyhow: this is nevertheless an improvement, guess we can therefore merge.

( @acolytec3 but maybe you also have a quick look and see if you agree here )